client/consts_p.go

```diff
@@ -37,6 +37,12 @@ const (
 	// operation is all but guaranteed to get them dropped.
 	UDPSleepSyncTime = time.Second
 
+	// Event log constants. These values limit the in-memory footprint
+	// of the event logging system.
+	EventLogExpiry         = 7 * 24 * time.Hour
+	EventLogLimitBytes     = 1e7
+	EventLogLineLimitBytes = 200
```

Comment:
maybe 500?

Reply:
I didn't have a good reason to choose 200 so happy to use 500